components/modal/index.en-US.md

```diff
@@ -0,0 +1,67 @@
+---
+type: Views
+category: Components
+english: Modal
+---
+
+Modal dialogs.
+
+## When to use
+
+When requiring users to interact with application, but don't want to jump to a new page to interrupt
+user's workflow, you can use `Modal` to create a new floating layer over currtent page for user 
+operations.
+
+In addition, if you need a simple confirm dialog to users, you can use `ant.Modal.confirm()`, 
```

Comment:
Additionally, if you need to show a simple confirm dialog, you can use `ant.Modal.confirm()`,